tests/network/cifs.pm

```diff
@@ -48,7 +48,10 @@ sub run {
     my $smb_remote = get_var("CIFS_TEST_REMOTE") // "currywurst.qam.suse.de";
     $self->select_serial_terminal;
     add_suseconnect_product(get_addon_fullname('phub')) if is_sle;    # samba-client requires package hub
-    zypper_call('in cifs-utils samba-client nmap');
+    my $ret = zypper_call('in cifs-utils samba-client nmap', exitcode => [0, 106]);
+    if ($ret == 106) {
+        record_soft_failure 'bsc#1152524 - [Build 18.1] openQA test fails whenever package hub repo is added: Valid metadata not found at specified URL';
```

Comment:
as long as the packages can not be installed due to the unavailability of phub before SLE15SP3 public beta, don't we need to abort the test and immediately return? Otherwise I would expect tests to fail afterwards anyway.

Reply:
It is not about the packages but repo data. The package seems to work.
https://openqa.suse.de/tests/4965893#step/cifs/25

Reply:
oh, that's better than :)

Good